Samples~/SimpleInteraction/SimpleInteraction.cs

```diff
@@ -16,11 +16,11 @@ void Start()
             playerText.Select();
         }
 
-        void onInputFieldSubmit(string message)
+        async void onInputFieldSubmit(string message)
         {
             playerText.interactable = false;
             AIText.text = "...";
-            _ = llmCharacter.Chat(message, SetAIText, AIReplyComplete);
+            await llmCharacter.Chat(message, SetAIText, AIReplyComplete);
```

Comment:
there is no need to await for the end of the Chat since the SetAIText will be anyway called asynchronously